### ISSUE TITLE
Need to update documentation 

### ISSUE THREAD

**Issue by vinaymavi** [potatolondon/djangae#966]
We need to update documentation https://djangae.readthedocs.io/en/latest/installation/#installation 
for gcloud SDK 

**Need to remove**
app.yaml: 
* application:
* version: 

as this is not required.

**Need to add**
Deployment command:
gcloud app deploy --project [GCLOUD_PROJECT_NAME]  ./

**Comment by seawolf42** [comment]
I would be happy to do this for you!

**Comment by adamalton** [comment]
Djangae is currently build to run with the standalone SDK.  But as Google seem to be pushing everything towards the `gcloud` SDK, we should probably support that.

Does `gcloud app deploy` not allow you to specify the `application` and `version` in app.yaml?  I.e. does it actually break if you leave them in?

For deploying, we can just provide both deployment commands (for `gcloud` and for the standalone SDK with `appcfg.py update`.

**Comment by seawolf42** [comment]
Correct, you can't supply the application and version in app.yaml. I've had everything running without GoogleAppEngineLauncher for over a month now, so I can write that up along with an example shell script that deploys using a local config file for some options and pulls the version from either local config or from the git version tag. I'll do that this weekend.

**Comment by seawolf42** [comment]
There should probably also be changes made to the [djangae-scaffold](https://github.com/potatolondon/djangae-scaffold) app to reflect the updates made here.
